### PR TITLE
Added usage to README.md

### PR DESCRIPTION
**Pull request by peroman86**
@shaharke 
I added usage to README.md

Just added that the required module must be a function

### CODE REVIEW

**Comment by peroman86** [comment]
@shaharke

**Comment by peroman86** [comment]
@shaharke , don't forget me too :)

File: README.md [hunk 1]
```diff
@@ -20,7 +20,22 @@ the execution of phases.
...
 ## Usage
 
-_TBD_
+    const bootleg = require('bootleg');
+
+    module.exports = function bootApp() {
+    const app = bootleg();
+
+    // pass the phase name and you're module that you want to execute.
+    // the module that you require need to bee wrapped with function, object module won't work.
```
Comment (shaharke) [inline README.md]: Fix type (bee = be)

**Review by shaharke — CHANGES_REQUESTED** [review]

**Comment by shaharke** [comment]
@peroman86 squash and you're done

**Comment by peroman86** [comment]
@shaharke , squashed

**Review by shaharke — APPROVED** [review]

**Pull request merged**